mvpoly/src/prime.rs

```diff
@@ -239,15 +242,26 @@ impl<F: PrimeField, const N: usize, const D: usize> MVPoly<F, N, D> for Dense<F,
                     let degree = naive_prime_factors(*idx, &mut prime_gen)
                         .iter()
                         .fold(0, |acc, (_, d)| acc + d);
-                    if degree > max_degree {
+                    if degree > max_degree || rng.gen_range(0..10) == 0 {
+                        // Adding zero coefficients one every 10 monomials
                         F::zero()
                     } else {
                         F::rand(rng)
```

Comment:
I'd simply replace this line with a generator biased toward 0

Reply:
> with a generator biased toward 0

How would you do that?

Reply:
generate a nb between 0-10, if that number is 10 return 0, otherwise generate uniformly a field elment

Reply:
But that's what the code does, doesn't?